bridge_adaptivity/config/settings/test.py

```diff
@@ -0,0 +1,27 @@
+# coding: utf-8
+
+from base import *
+
+TEST_RUNNER = 'config.test_runner.PytestTestRunner'
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': 'test.db',
+    }
+}
+
+SECRET_KEY = 'KEY'
+
+SOCIAL_AUTH_TWITTER_KEY = 'test_key'
```

Comment:
Are these needed?

Reply:
We don't need these settings.
I will remove it

Reply:
## About the tests.

### if you locally run project in docker then you can do this:

```
docker-compose -f docker-compose_local.yml up -d 
# first command will build and run container BFA_local
docker exec -it BFA_local python manage.py test
# second command will run tests inside of this container
```

### From other side if you don't use docker and just run project on your local machine:
```
pip install requirements_local.txt
python manage.py test
# or you can use next command to run tests. they are equal
pytest
```